File: 프로그래머스/unrated/181939. 더 크게 합치기/더 크게 합치기.cpp
#include <string>
#include <vector>
#include<cstdio>

using namespace std;

int solution(int a, int b) {
    int answer = 0;
    
    string A = to_string(a) + to_string(b);
    string B = to_string(b) + to_string(a);
    
    answer = stoi(A) > stoi(B) ? stoi(A): stoi(B);
    
    return answer;
}

File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.py
def solution(a, b):
    answer = 0
    
    a_b = str(a)+str(b)
    b_a = str(b)+str(a)
    
    if a_b >= b_a:
        answer = a_b
    else:
        answer = b_a
    
    answer = int(answer)
    return answer